src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -2402,11 +2414,11 @@ public long GetRevisionsCount(DocumentsOperationContext context, Slice id)
 
         public (Document[] Revisions, long Count) GetRevisions(DocumentsOperationContext context, string id, long start, long take)
         {
-            using (DocumentIdWorker.GetSliceFromId(context, id, out Slice lowerId))
-            using (GetKeyPrefix(context, lowerId, out Slice prefixSlice))
-            using (GetLastKey(context, lowerId, out Slice lastKey))
```

Comment:
Just to make sure, the other usages of `GetLastKey()` are okay, right? No need to replace them by `GetKeyWithEtag()`?

Reply:
It seems to do the same:
![image](https://github.com/user-attachments/assets/62f6b917-c243-40b0-b37b-de4e774b4233)
I reverted this change

Reply:
OK